new/LIT7006Yezarreu.xml

```diff
@@ -7,9 +7,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">ይዘርኡ፡ ዘርአ፡ መስተገብራን፡ እለ፡ ይነብሩ፡ ሐቅለ፡ ወየሀድሩ፡ ገዳመ። </title>
                 <title xml:lang="gez" type="normalized" corresp="#t1">Yǝzarrǝʾu zarʾa mastagabrān ʾǝlla yǝnabbǝru ḥaqla 
-                    wa-yahaddǝru gadāma, ... (Qǝne of mawaddǝs type)</title>
-                <title xml:lang="en" corresp="#t1">The laborers are sowing seeds, those who dwell in the field and sleep in the wilderness,...
-                    , ...</title>
+                    wa-yahaddǝru gadāma ... (Qǝne of mawaddǝs type)</title>
```

Comment:
Perhaps it would be good to refer to the authority file for mawaddǝs-type qǝnes somewhere in the file? Sorry if I'm missing it

Reply:
I think, this is not our practice to do this in the title, because we do not want to have links in the title headline.

Reply:
Then maybe at another place in the file? Sorry, again, if it's there and I'm missing it

Reply:
So the keywords and the authority files fulfil the same function here?

Reply:
At least, this is, how I understand it.

Reply:
In fact, no. Keywords (meaning exposed in the authority files) mark the thematics which is not necessarily reflected clearly in the source; what I mark in the description is the usage of foreing terms.

Reply:
Actually, you may be right. I will check how the last qene records were encoded. What I dislike and what I found distracting as there is a huge amount of code within very short text.